src/test/resources/uk/ac/ebi/embl/api/validation/file/template/ERT000002-rRNA-with-sample.tsv.gz.expected

```diff
@@ -0,0 +1,48 @@
+ID   XXX; SV 1; linear; genomic DNA; STD; HUM; 340 BP.
+XX
+ST * private
+XX
+AC   ;
+XX
+AC * _1
+XX
+XX
+DE   Homo sapiens mitochondrion partial 16S rRNA gene, clone 24
+XX
+KW   .
+XX
+OS   Homo sapiens (human)
+OC   Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi; Mammalia;
```

Comment:
- The OC line should NOT be needed at this point yet. The logic is that we FIRST contrust a file file and then parse it into the data model. The OC line will be created as a side effect of parsing th flat file fully and from having full taxon/lineage information,

Reply:
In case of sample we are getting taxon for the respective sample and setting the taxon to the sourceFeature `sourceFeature.setTaxon(taxon);` The taxon contains semicolon separated  lineage (Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi; Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini; Catarrhini; Hominidae; Homo; ) That is in turn added in flat file